src/Monitoring/LoopStatistics.cpp

```diff
@@ -197,6 +197,14 @@ template <> struct type2nc<int32_t> {
 template <> struct type2nc<uint32_t> {
     static constexpr auto type = NC_UINT;
 };
+
+// Long int is guaranteed to be of size 64 on Unix/Linux:
+// https://en.cppreference.com/w/cpp/language/types
+// This specialization is required on some architectures as it is later used for the type of
+// timespec::tv_sec, which is not fixed in the C++ standard.
+template <> struct type2nc<long int> {
```

Comment:
However, if `std::is_same_v<long int, int64_t>` holds, we got a re-definition. Since `int64_t` is merely a type redefinition. For some suggestions, we could either

* `#ifdef __APPLE__` that one.
* choose the type generically based on type traits and size (`std::numeric_limits` etc may help as well)
* if NetCDF has a HDF5-like `H5T_NATIVE_LONG`, then that would be also a way to go: do it for `int`, `long`, `long long` and you should be good.

Reply:
Huh, that's actually a good point. The CI catched it anyway. I'll look for a nice solution :)